.github/workflows/build.yml

```diff
@@ -14,14 +14,20 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-java@v3
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Set up Gradle
+        uses: gradle/actions/setup-gradle@v4
+
+      - name: Set up Java
+        uses: actions/setup-java@v4
         with:
           distribution: 'temurin'
           java-version: '21'
 
-      - run: chmod +x gradlew
-
       - name: Set up Node JS
         uses: actions/setup-node@v3
```

Comment:
Must be `v4`.

Reply:
will be done after this pull request as minegame will be reworking the publishing process